doc/classes/RandomNumberGenerator.xml

```diff
@@ -17,6 +17,22 @@
 		<link title="Random number generation">$DOCS_URL/tutorials/math/random_number_generation.html</link>
 	</tutorials>
 	<methods>
+		<method name="rand_weighted">
+			<return type="int" />
+			<param index="0" name="weights" type="PackedFloat32Array" />
+			<description>
+				Returns a random index with non-uniform weights. Prints an error and returns [code]null[/code] if the array is empty.
```

Comment:
This actually returns `0` now.
```suggestion
				Returns a random index with non-uniform weights. Prints an error and returns [code]0[/code] if the array is empty.
```
But I'm questioning, would it make sense to return `0` or `-1`?